gulpfile.babel.js

```diff
@@ -72,9 +72,16 @@ gulp.task('run:jsonlint', () => gulp.src(['**/*.json', '!node_modules/**'])
   .pipe(jsonlint.failAfterError())
 );
 
-gulp.task('run:test', ['build:test'], () => {
+gulp.task('jest', function() {
+  return gulp.src('src/**/__tests__/browser').pipe(jest({
```

Comment:
Below would run the task without __tests__ directory.

gulp.task('run:test-client', ['build:test'], function() {
  return gulp.src('dist-test').pipe(jest({
    config: {
      rootDir: 'dist-test/test/browser',
      testRegex: '/index\\.js'
    },
    unmockedModulePathPatterns: ['react']
  }));
});